src/execution.js

```diff
@@ -86,3 +101,92 @@ export function getData(projectId, elements, executionConfiguration) {
     return d.promise();
 }
 
+const getFilterExpression = listAttributeFilter => {
+    const attributeUri = get(listAttributeFilter, 'listAttributeFilter.attribute');
+    const elements = get(listAttributeFilter, 'listAttributeFilter.default.attributeElements', []);
```

Comment:
What if elements/elementsForQuery are empty ? We'll get invalid MAQL condition [attribute uri] NOT IN/IN ().